tests/integ/test_udf.py

```diff
@@ -1584,3 +1584,24 @@ def test_udf_pickle_failure(session):
         "local environment first, add it to the UDF with session.add_import(), and read it from "
         "the UDF." in str(ex_info)
     )
+
+
+def test_comment_in_udf_description(session):
```

Comment:
Is it possible it run all current udf tests with source_code mode? Though it will be time-consuming...

Reply:
yes, since we're now launching the beta release plan for experimental features, I think that would a better place for real execution.